popup.js

```diff
@@ -11,13 +11,32 @@ function execute_inline_code(code_to_execute, notification_text) {
             s=s.slice(0, 29);
             if(done==false && s.localeCompare("https://www.youtube.com/watch")==0){
                 found = tabs[i].id;
+                
+                
+
                 chrome.tabs.executeScript( found, {
                     code: code_to_execute
                 }, function callBack(results) {
                     paused = results;
                 });
                 if(notification_text.localeCompare('')!=0) {
-                    send_notification(notification_text);
+
+                    chrome.tabs.get(found,function(tab) {
+                        send_notification(notification_text + "\n" + tab.title);
+                    });
+
+                    
+                }
+                else
+                {
+                    var delay=3000; //3 second
```

Comment:
Added this delay for prev and next tracks as time is required for loading new page.